pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -260,14 +260,25 @@ func (se *sumologicexporter) pushLogsData(ctx context.Context, ld pdata.Logs) er
 
 				currentMetadata = sdr.filter.filterIn(attributes)
 
+				// Extract attributes which should be send as json body
+				// and connect them with log record
+				lp := logPair{
+					log:            log,
+					bodyAttributes: sdr.filter.filterOut(attributes),
+				}
+
 				if se.config.TranslateAttributes {
-					translateAttributes(attributes)
+					// Translate fields so they can be properly send in JSON format
+					translateAttributes(lp.bodyAttributes.orig)
+					// Translate metadata co it can be used along with translated source host templates
 					translateAttributes(currentMetadata.orig)
+					// Translate original log attribute in case of using OTLP format
```

Comment:
> in case of using OTLP format

... but it's done regardless of configured format? So maybe make it conditional? Or change the comment?

Reply:
I make it conditional, did the same for JSON, but honestly not sure if this is not an overengineering